CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ Major Changes
 -------------
 
 - All modules now support SSL over HTTPS and socket level timeout.
+- HTTPS SSL certificate validation is a ``breaking change`` and needs an update to the playbooks.
```

Comment:
IMHO, it would be good to refer users to the README section, something like below:

**HTTPS SSL certificate validation is a breaking change and will require modification in the existing playbooks. Please refer to [SSL Certificate Validation](https://github.com/dell/dellemc-openmanage-ansible-modules#SSL-Certificate-Validation)**